src/client/stylesheets/components/_autocomplete.scss

```diff
@@ -0,0 +1,7 @@
+.autocomplete__wrapper {
+  width: 300px;
```

Comment:
wondering if we could try to use any of the existing govuk classes to avoid hardcoding a value here (thinking about small screens for example)